openlibrary/catalog/marc/parse.py

```diff
@@ -358,7 +358,6 @@ def read_original_languages(rec: MarcBase) -> list[str]:
     found = []
     fields = rec.get_fields('041')
     for f in fields:
-        is_translation = f.ind1() == '1'
```

Comment:
This is probably an indication we should be using this data for something rather than just losing it...

Reply:
Could be. I dug around and couldn't find it ever being used since first added a few years ago https://github.com/internetarchive/openlibrary/pull/7407

Reply:
I'm not sure it adds any information. If 041$a (language) and 041$h (original language) are different, it's a translation. If 041 first indicator = 1 saying it's a translation, it's either redundant with the other information, or, if e.g. 041$h is missing, doesn't provide any actionable info, so I'm OK with removing it.